app/components/search-results/row.js

```diff
@@ -0,0 +1,44 @@
+import Component from '@ember/component';
+import { get, computed } from '@ember/object';
+import { alias, or } from '@ember/object/computed';
+
+export default Component.extend({
+  kind: alias('item.kind'),
+  imageUrl: or('item.avatar', 'item.posterImage'),
+  title: or('item.name', 'item.canonicalTitle'),
```

Comment:
might want to use `computedTitle` here due to our preference options.

Reply:
Can't use `computedTitle` because these are Algolia JSON instead of actual ED models